grid-proxy/pkg/types/types.go

```diff
@@ -79,6 +79,8 @@ type Limit struct {
 	Page      uint64
 	RetCount  bool
 	Randomize bool
+	SortBy    string
+	SortOrder string
```

Comment:
I don't like that there is absolutely no validation on the values of these and then are immediately used in sql statements. (if there are any validation please point me to it)

Beside sql injection, accepting any value as input from the user is generally wrong.

Instead, i would define 2 types 
```go
type SortBy string

const (
   AscSort SortBy = "ASC"
   DescSort SrotBy = "DESC"
)

func(s SortBy) Valid()error {
   // if the s value is not exactly one of the possible values return an error.
}
```

Then the `limit` structure should have it's own Valid() method that validate the input AND call valid() on any internal attribute that needs validation.

Then the Limit structure need to always be validate the moment it's parsed from user input. (query params)